support-workers/.history

```diff
@@ -0,0 +1,4 @@
+project support-workers
+project
+support-workers
+exit
```

Comment:
What is this file?  Is it meant to be checked in?

Reply:
this should be deleted - I think paul must have run `sbt` from the subfolder by mistake